core/src/main/java/org/jdbi/v3/core/result/ResultProducers.java

```diff
@@ -49,16 +58,30 @@ private ResultProducers() {
      * @return ResultBearing of result rows.
      * @see PreparedStatement#getResultSet()
      */
+    @Deprecated
     public static ResultProducer<ResultBearing> returningResults() {
+        return new ResultProducers(false).returningQueryResults();
+    }
+
+    /**
+     * Result producer that returns a {@link ResultBearing} over the statement result rows.
+     *
+     * @return ResultBearing of result rows.
+     * @see PreparedStatement#getResultSet()
+     */
+    public ResultProducer<ResultBearing> returningQueryResults() {
```

Comment:
Just seeing this. This method and `getResultSet` could be static by getting the config through the `StatementContext`.

Reply:
@stevenschlansker ^

Reply:
Fixing it in a followup commit, this means we don't have to change the api at all, thanks!

Reply:
:+1: